php_phongo.c

```diff
@@ -1638,30 +1613,16 @@ static void php_phongo_cursor_iterator_rewind(zend_object_iterator *iter TSRMLS_
 {
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 	php_phongo_cursor_t        *cursor = cursor_it->cursor;
+	const bson_t               *doc;
 
 	php_phongo_cursor_free_current(cursor);
 	cursor_it->current = 0;
```

Comment:
I don't think so. `cursor_it->current` is zero from "get_iterator" allocating the `php_phongo_cursor_iterator` struct. The "move_forward" handler isn't called until we advance to the element beyond the first, since only "rewind" would be called (obviously after "get_iterator") when iterating through the first element in a `foreach`.